extensions/functions_string.yaml

```diff
@@ -51,6 +51,13 @@ scalar_functions:
     description: >-
       Extract a substring of a specified `length` starting from position `start`.
       A `start` value of 1 refers to the first characters of the string.
+
+      The `negative_start` option applies to the `start` parameter. `WRAP_FROM_END` means
+      the returned substring will start from the end of the `input` and move backwards.
```

Comment:
Maybe "means the index will start from the end..."?  It still kind of sounds like the string itself is moving backwards.

Reply:
Thanks for the catch.  updated!